cmd/agent/main.go

```diff
@@ -101,6 +105,7 @@ func (o *GrpcProxyAgentOptions) Flags() *pflag.FlagSet {
 	flags.DurationVar(&o.syncInterval, "sync-interval", o.syncInterval, "The initial interval by which the agent periodically checks if it has connections to all instances of the proxy server.")
 	flags.DurationVar(&o.probeInterval, "probe-interval", o.probeInterval, "The interval by which the agent periodically checks if its connections to the proxy server are ready.")
 	flags.StringVar(&o.serviceAccountTokenPath, "service-account-token-path", o.serviceAccountTokenPath, "If non-empty proxy agent uses this token to prove its identity to the proxy server.")
+	flags.StringVar(&o.agentAddress, "agent-addr", o.agentAddress, "Addresses that are reachable through this agent. e.g.,host=localhost,cidr=127.0.0.1/16,ipv6=:::::,failure-zone=us-central1-b.")
```

Comment:
Might be worth documenting you support multiple values by repeating keys. Eg host=localhost,host=node1.mydomain.com